src/js/common/schemaform/actions.js

```diff
@@ -158,9 +158,10 @@ export function submitForm(formConfig, form) {
 
     return promise
       .then(resp => dispatch(setSubmitted(resp)))
-      .catch(error => {
+      .catch(errorReceived => {
         // overly cautious
-        const errorMessage = _.get('message', error);
+        const error = errorReceived instanceof Error ? errorReceived : new Error(errorReceived);
```

Comment:
Everything downstream expects this to be of type `Error`. If it's `undefined`, a `string`, or an `object` this will force it something that won't blow up.